PowerEditor/src/Parameters.h

```diff
@@ -875,7 +875,7 @@ struct NppGUI final
 	generic_string _uriSchemes = TEXT("svn:// cvs:// git:// imap:// irc:// irc6:// ircs:// ldap:// ldaps:// news: telnet:// gopher:// ssh:// sftp:// smb:// skype: snmp:// spotify: steam:// sms: slack:// chrome:// bitcoin:");
 	NewDocDefaultSettings _newDocDefaultSettings;
 
-	generic_string _dateTimeFormat = TEXT("Y-M-D h:m:s");
+	generic_string _dateTimeFormat = TEXT("yyyy-MM-dd HH:mm:ss");

```

Comment:
It would be nice if it was possible to update the value already saved to config XML.